src/test/resources/application.conf

```diff
@@ -1,2 +1,2 @@
-key="7f07eabdc97e9555e6cfc283a3a3fb9e"
-token="ATTA1418663ae4e3999c60e6d04258e61619af886960da297bf6709f56d1e631ff1084784FA0"
```

Comment:
the same problem 
change token and key to the previous

Reply:
I can't run the tests with the previous token and key